### PR TITLE
hotfix: readme formating

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

The introduced spacing at the end of the line violates formatting and prevents any PR from landing, so suggesting to have each sentence on a separate line which I believe also increases readability...

<details>
  <summary><b>Before submitting</b></summary>

- Was this **discussed/agreed** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- Did you make sure to **update the documentation** with your changes? (if necessary)
- Did you write any **new necessary tests**? (not for typos and docs)
- [x] Did you verify new and **existing tests pass** locally with your changes?
- Did you list all the **breaking changes** introduced by this pull request?
- Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

</details>

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

<details>
  <summary>Reviewer checklist</summary>

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

</details>

cc @borda

### CODE REVIEW

**Review by awaelchli — APPROVED** [review]
Needs @williamFalcon 😄

File: README.md [hunk 1]
```diff
@@ -95,8 +95,8 @@ ______________________________________________________________________
 
 ## Lightning has 3 core packages
 
-[PyTorch Lightning: Train and deploy PyTorch at scale](#pytorch-lightning-train-and-deploy-pytorch-at-scale).    
-[Lightning Fabric: Expert control](#lightning-fabric-expert-control).    
+[PyTorch Lightning: Train and deploy PyTorch at scale](#pytorch-lightning-train-and-deploy-pytorch-at-scale).
+[Lightning Fabric: Expert control](#lightning-fabric-expert-control).
 [Lightning Apps: Build AI products and ML workflows](#lightning-apps-build-ai-products-and-ml-workflows).
```
Comment (Borda) [inline README.md:100]:
```suggestion
&nbsp; &nbsp; &nbsp; &nbsp;
[Lightning Fabric: Expert control](#lightning-fabric-expert-control).
&nbsp; &nbsp; &nbsp; &nbsp;
[Lightning Apps: Build AI products and ML workflows](#lightning-apps-build-ai-products-and-ml-workflows).
```

File: README.md [hunk 2]
```diff
@@ -95,8 +95,10 @@ ______________________________________________________________________
 
 ## Lightning has 3 core packages
 
-[PyTorch Lightning: Train and deploy PyTorch at scale](#pytorch-lightning-train-and-deploy-pytorch-at-scale).    
-[Lightning Fabric: Expert control](#lightning-fabric-expert-control).    
+[PyTorch Lightning: Train and deploy PyTorch at scale](#pytorch-lightning-train-and-deploy-pytorch-at-scale).
+&nbsp; &nbsp; &nbsp; &nbsp;
+[Lightning Fabric: Expert control](#lightning-fabric-expert-control).
+&nbsp; &nbsp; &nbsp; &nbsp;
```
Comment (Borda) [inline README.md:101]:
```suggestion
<br/>
[Lightning Fabric: Expert control](#lightning-fabric-expert-control).
<br/>
```

**Review by carmocca — APPROVED** [review]

**Comment by carmocca** [comment]
CI is deadlocked: this PR needs https://github.com/Lightning-AI/lightning/pull/17134 and https://github.com/Lightning-AI/lightning/pull/17134 needs this PR.

One will have to be force merged.

**Comment by Borda** [comment]
> CI is deadlocked: this PR needs #17134 and #17134 needs this PR.

maybe typo, you refer to the same pr twice

**Review by lantiga — APPROVED** [review]

**Pull request merged**